pg_backend_support.c

```diff
@@ -49,7 +49,7 @@ extern MemoryContext plr_caller_context;
 /*
  * static declarations
  */
-static char *get_lib_pathstr(Oid funcid);
+static char *get_lib_pathstr(Oid langOid);
```

Comment:
there are actually oid's for languages..

Reply:
Now it is for language. It used to be for function. I had to do that as we do not have `funcid` for inline handler.